content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -47,15 +47,10 @@ resource pressure.
 Changing the settings such that your container runtime and kubelet use `systemd` as the cgroup driver
 stabilized the system. To configure this for Docker, set `native.cgroupdriver=systemd`.
 
-{{< caution >}}
-Changing the cgroup driver of a Node that has joined a cluster is strongly *not* recommended.  
-If the kubelet has created Pods using the semantics of one cgroup driver, changing the container
-runtime to another cgroup driver can cause errors when trying to re-create the Pod sandbox
-for such existing Pods. Restarting the kubelet may not solve such errors.
-
-If you have automation that makes it feasible, replace the node with another using the updated
-configuration, or reinstall it using automation.
-{{< /caution >}}
+### Migrating to the `systemd` driver
+
+Follow the [Migration guide](/tasks/administer-cluster/kubeadm/configure-cgroup-driver)
```

Comment:
i agree with the point that the kubelet if not run with systemd does not need to use the systemd driver.

but `container-runtimes.md` is really about the kubelet package (DEB/RPM) that k8s distributes and that package installs the kubelet as a systemd service, which means it should use the systemd driver.

also all the container runtimes on the page include steps to setup using the systemd driver.

Reply:
> So I would suggest to keep existing content (which is still true and useful) and add a section called migration to systemd cgroup driver using kubeadm to make it explicit.

the change on this page is removing a `{{< caution >}}` blob about cgroup driver migration and replacing it with a guide for that. what other existing content do you mean?

Reply:
>what other existing content do you mean?
I mean the existing `{{<caution >}}` blob.

>also all the container runtimes on the page include steps to setup using the systemd driver.

I concerned more about pointing the instruction to a `kubeadm` page than assuming `systemd` is used. In reality, as far as I know, most setup do one systemd; but not all of them are using kubeadm for the bootstrap.

Reply:
ok, i see what you mean.
i can preserve the caution but rephrase it a little to say that it is a "disruptive operation" instead of saying "not recommended", because it's actually recommended.

also rephrase the new section to explain that this is a kubeadm guide:

```
### Migrating to the `systemd` driver in kubeadm clusters

Follow the [Migration guide](/tasks/administer-cluster/kubeadm/configure-cgroup-driver)
if you wish to migrate to the `systemd` cgroup driver in existing kubeadm clusters.
```

Reply:
perfect. thanks.
and maybe "%s/kubeadm clusters/kubeadm managed clusters/g"